docker/Dockerfile.mediamtx

```diff
@@ -1,11 +1,18 @@
-FROM ubuntu:20.04
+FROM ubuntu:24.04
 
-RUN apt update
-RUN apt install -y curl
+ARG TARGETARCH
+ENV MEDIAMTX_VERSION="v1.9.3"
 
-RUN curl -L https://github.com/bluenviron/mediamtx/releases/download/v1.9.3/mediamtx_v1.9.3_linux_amd64.tar.gz -o /mediamtx.tar.gz
-RUN tar xzvf /mediamtx.tar.gz
-RUN rm /mediamtx.tar.gz
+RUN apt-get update \
+    && apt-get install -y \
+      ca-certificates \
+      curl \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN curl -L https://github.com/bluenviron/mediamtx/releases/download/${MEDIAMTX_VERSION}/mediamtx_${MEDIAMTX_VERSION}_linux_amd64.tar.gz -o /mediamtx.tar.gz \
+    && tar xzvf /mediamtx.tar.gz \
+    && rm /mediamtx.tar.gz
 
 COPY docker/mediamtx.yml /mediamtx.yml
```

Comment:
can set the context of docker build to be inside `docker/` if there are no other files from the repo that are needed.